src/NServiceBus.AcceptanceTesting/Support/ScenarioConfigSource.cs

```diff
@@ -70,15 +70,6 @@ public T GetConfiguration<T>() where T : class, new()
                 }
             }
 
-            if (type == typeof(Logging))
```

Comment:
I think this means we have to set the standard level to WARN in the default server template